spec/models/delete_tag_filter_spec.rb

```diff
@@ -52,7 +52,7 @@ def new_add_filter(tag_name = 'just-a-tag')
         it 'returns all tag "a" taggings' do
           taggings = ActsAsTaggableOn::Tagging
                      .where(context: @hub.tagging_key, tag_id: @tag.id)
-          expect(@filter.deactivates_taggings).to match_array(taggings)
+          expect(@filter.deactivates_taggings(@filter.scope.taggable_items.pluck(:id))).to match_array(taggings)
```

Comment:
Why does this need to be explicit throughout the whole suite now? We no longer trust the filter to know all the items in its scope?

Reply:
The `deactivates_taggings` was updated without updating the test suite here https://github.com/berkmancenter/tagteam/commit/6b492121933c265794e481c9cc718f99655e418b#diff-2199655b9de6a8389bb982e331a05287L84.

Reply:
Do we know why it was changed? This seems like it's making the API worse. If we can't figure out why, I guess it's fine to keep it in as its relatively minor.

Reply:
@jdcc 

> Do we know why it was changed?

Looks like just some code refactoring.

> This seems like it's making the API worse

Agree.

> If we can't figure out why, I guess it's fine to keep it in as its relatively minor.

Yup!